bundlewrap/cmdline/generate_completions.py

```diff
@@ -4,8 +4,8 @@
 def bw_generate_completions(repo, args):
     compl_file = join(repo.path, '.bw_shell_completion_targets')
 
-    targets = [node.name for node in repo.nodes]
-    targets.extend([group.name for group in repo.groups])
+    targets = [f'node:{node.name}' for node in repo.nodes]
+    targets.extend([f'group:{group.name}' for group in repo.groups])
```

Comment:
Bundlewrap should probably also support this selector: https://github.com/bundlewrap/bundlewrap/issues/827